src/classes/components/mechskills/MechSkillsController.ts

```diff
@@ -48,9 +48,7 @@ class MechSkillsController {
 
   //TODO: requires target change as part of bonus rework
   public get MaxHASEPoints(): number {
-    const p = this.Parent as any
-    if (!p.Level) return Rules.MinimumMechSkills
```

Comment:
If we decide that `this.Parent` should *stay* as type `IHASEContainer`, we will just need to modify this line so that `0` is accounted for appropriately, whether that be checking for a `number` type, or for whether p.Level is `undefined` or `null`, or even simply just performing the `Bonus.Int()` function without `p.Level` included.

Reply:
I think this is a good change -- NPCs *might* implement IHASEContainer in the upcoming GM rebuild but I haven't totally settled on it yet, but as you've pointed out it's pretty easy to address this when the time comes